app/api/ms_api_utils.py

```diff
@@ -28,12 +28,22 @@ def post_request(request_url, data):
             request_url, json=data, headers={"Accept": "application/json"}
         )
         response.raise_for_status()
-    except requests.ConnectionError as e:
-        return f"{e.response.json()}", HTTPStatus.INTERNAL_SERVER_ERROR
-    except requests.HTTPError as e:
-        return f"{e.response.json()}", e.response.status_code
+        response_message = response.json()
+        response_code = response.status_code
+    except requests.exceptions.ConnectionError as e:
+        response_message = messages.INTERNAL_SERVER_ERROR
+        response_code = json.dumps(HTTPStatus.INTERNAL_SERVER_ERROR)
+        logging.warning(f"{e}")
+        return response_message, response_code
+    except requests.exceptions.HTTPError as e:
+        response_message = e.response.json()
+        response_code = e.response.status_code
+        return response_message, response_code
     except Exception as e:
```

Comment:
Same as my previous comment on catching exceptions

Reply:
We need to fix this beyond just logging it. If it's a specific message it won't be checked.
And this should not be a logging.warning, it should be fatal

Reply:
Changed it to logging.fatal now. 👍

Reply:
@ramitsawhney27 Any tips what else need to be done here?

Reply:
Resolved for now